specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2022-03-01-preview/activeDirectoryConnectors.json

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
nit - if possible name your file according to its content, e.g. activeDirectoryConnectorsDefinition.json

Reply:
At the present moment, following other RP's such as `Microsoft.Sql`, our file naming convention is that the file is named after the resource type

Reply:
Yeah, but there's one more layer - resource type spec vs definition which in this case is not easy to distinguish.

Reply:
Keeping it as is for now, but definitely going to consider renaming files in a future update.